<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticFlight</class>
 <widget class="QWidget" name="StatisticFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>910</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Статистика</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">Steel=rgb(0, 0, 127)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="lb_nameAiroport">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Аэропорты :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="cb_listNameAiroport">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cb_listMonth">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lb_month">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Месяцы :</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>800</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pb_StaticYear">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Вывести статистику за год</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QPushButton" name="pb_close">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
